packages/ckeditor5-engine/src/view/observer/focusobserver.js

```diff
@@ -37,7 +37,10 @@ export default class FocusObserver extends DomEventObserver {
 			// overwrite new DOM selection with selection from the view.
 			// See https://github.com/ckeditor/ckeditor5-engine/issues/795 for more details.
 			// Long timeout is needed to solve #676 and https://github.com/ckeditor/ckeditor5-engine/issues/1157 issues.
-			this._renderTimeoutId = setTimeout( () => view.forceRender(), 50 );
+			//
+			// Changing document.isFocused marks view as changed since last rendering, it would be best if view got rendered
+			// on the next selectionChange event, but as a fallback let's trigger rendering in 50ms.
```

Comment:
```suggestion
			// Using `view.change()` instead of `view.forceRender()` to prevent double rendering
			// in a situation where `selectionchange` already caused selection change.
```

Reply:
Is this comment true and better?